app/controllers/dogs_controller.rb

```diff
@@ -117,130 +115,112 @@ def destroy
   end
 
   def switch_view
-    if !session[:mgr_view]
-      session[:mgr_view] = true
-    else
-      session[:mgr_view] = false
-    end
+    session[:mgr_view] = !session[:mgr_view]
 
     redirect_to dogs_path
   end
 
-
   private
 
-    def dog_params
-      params.require(:dog).permit(  :name,
-                                    :tracking_id,
-                                    :primary_breed_id,
-                                    :primary_breed_name,
-                                    :secondary_breed_id,
-                                    :secondary_breed_name,
-                                    :status,
-                                    :age,
-                                    :size,
-                                    :is_altered,
-                                    :gender,
-                                    :is_special_needs,
-                                    :no_dogs,
-                                    :no_cats,
-                                    :no_kids,
+  def dog_params
```

Comment:
I'm not a fan of indenting private methods twice so the following all looks like it was modified but was mostly outdented 2 spaces.

https://github.com/ophrescue/RescueRails/pull/459/files?w=1 to see without whitespace changes